pkg/controller/webhook/webhook.go

```diff
@@ -89,6 +108,41 @@ func notify(destination string, message []byte) error {
 		destination, resp.Status)
 }
 
+func notifyWS(destination string, message []byte) error {
+	ctx, cancel := context.WithTimeout(context.Background(), notificationSendTimeout)
```

Comment:
parent ctx should be an argument to Notify.

Reply:
Added a follow up TODO and created an issue, applies to HTTP as well.